test/e2e/mutate/mutate_test.go

```diff
@@ -511,7 +511,7 @@ func Test_Mutate_Existing(t *testing.T) {
 			Expect(err).NotTo(HaveOccurred())
 
 			// wait for UR to be completed
-			time.Sleep(3 * time.Second)
+			time.Sleep(20 * time.Second)
```

Comment:
Does it take 20s to apply a single mutate rule? That doesn't sound right..

Reply:
It's not,  tried to run the test locally is working in 3sec, but here in github action it's taking time